lib/generation/generators/visual-widgets/pb_padding_gen.dart

```diff
@@ -48,15 +48,15 @@ class PBPaddingGen extends PBGenerator {
       }
       if (value != null) {
         buffer.write(
-            '$position: ${relativePadding(source.generator.templateStrategy, isVertical, value)},');
+            '$position: ${relativePadding(source.currentContext.treeRoot.rootNode.generator.templateStrategy, isVertical, value)},');
```

Comment:
Hey @GVTopCoder, is there a reason we are switching the generator from the `source` to the `rootNode`? 🤔

Reply:
yea, the node is below the symbol instance and the root template strategy
is what is used to identify layout.  it doesn’t change the generator